pkg/skaffold/build/tag/git_commit.go

```diff
@@ -75,7 +75,8 @@ func (c *GitCommit) Labels() map[string]string {
 func (c *GitCommit) GenerateFullyQualifiedImageName(workingDir string, imageName string) (string, error) {
 	ref, err := c.makeGitTag(workingDir)
 	if err != nil {
-		logrus.Warnln("Unable to find git commit:", err)
```

Comment:
Maybe we can add: "Re-run with --v=debug for more details" ?

Reply:
I don't know about that. Should we tell the user to run in debug mode each time we print a warning?